src/Psalm/Internal/Analyzer/ProjectAnalyzer.php

```diff
@@ -811,14 +849,34 @@ public function setPhpVersion(string $version)
 
     /**
      * @param array<string, bool> $issues
+     * @throws UnsupportedIssueToFixException
      *
      * @return void
      */
     public function setIssuesToFix(array $issues)
     {
+        $supported_issues_to_fix = static::getSupportedIssuesToFix();
+
+        $unsupportedIssues = array_diff(array_keys($issues), $supported_issues_to_fix);
+
+        if (! empty($unsupportedIssues)) {
+            throw new UnsupportedIssueToFixException(
+                'Psalm doesn\'t know how to fix issue(s): ' . implode(', ', $unsupportedIssues) . PHP_EOL
+                . 'Supported issues to fix are: ' . implode(', ', $supported_issues_to_fix)
```

Comment:
Can't be pasted into the command line as above. This is better:
```suggestion
                . 'Supported issues to fix are: ' . implode(',', $supported_issues_to_fix)
```